### DISCUSSION TITLE
Startup crash

### DISCUSSION THREAD

**Discussion by TomGoat98** [leoetlino/project-restoration#168] [General]
I have just installed the 1.5.8 patch (the most recent at time of writing), and the game has crashed on start up on my 3DS running Luma.

Just simply starting the game has caused the crashes for me, with the correct installation instructions. 

The Arm11 dumps have been attached.

*EDIT* - I am running the cartridge version of the game patched up to 1.1

[MM3DPR 1.5.8 Crash dumps.zip](https://github.com/leoetlino/project-restoration/files/4911754/MM3DPR.1.5.8.Crash.dumps.zip)

**Comment by leoetlino** [comment]
Which version of the patch are you using? Are you using the latest Luma?

**Comment by TomGoat98** [comment]
Latest version of luma and the patch

**Comment by leoetlino** [comment]
Are you using v100, v101 or v110? Have you tried all three variants of the patch? Make sure you delete the <TITLE_ID> folder between each attempt

**Comment by TomGoat98** [comment]
I have used all three versions of the patch, deleting the folder with each attempt, to no avail

**Comment by leoetlino** [comment]
Hmm, v100 does work for me, so I'm not sure what's wrong...

This should make no difference at all, but can you try all three variants again **after uninstalling the 1.1 update**? If that's possible, anyway.
